pkg/ovs/ovsconfig/ovs_client.go

```diff
@@ -670,3 +749,15 @@ func (br *OVSBridge) GetOVSVersion() (string, Error) {
 
 	return res[0].Rows[0].(map[string]interface{})["ovs_version"].(string), nil
 }
+
+func (br *OVSBridge) GetBridgeName() string {
+	return br.name
+}
+
+func (br *OVSBridge) SetUplinkName(uplikName string) {
+	br.uplinkName = uplikName
```

Comment:
Why we need to save uplink name in bridge? I think no other cases save state (passed from caller) to a bridge.

Reply:
Yes, _SetUplinkName_ is only used on windows paltform when prepare OVS bridge. There's no way to save this state, so I add the method.

Reply:
Could we just pass it to any func that creates/updates the uplink?

Reply:
I meant now all state saved in Bridge is from OVSDB. Better to keep a single source, otherwise we wont easily know where the state is from.

Reply:
Correct. I save the uplink name in CreateUplinkPort().
func CreateUplinkPort() {
    br.uplinkName = ifDev
    return br.CreatePort(...)
}